frontend/src/app/output/navigate/navigate-content/navigate-content.component.spec.ts

```diff
@@ -1,109 +0,0 @@
-import {async, ComponentFixture, TestBed} from '@angular/core/testing';
```

Comment:
Maybe keep at least a test to show it can be created, even if the component is now almost empty

Reply:
Couldn't make it work in the test due to import missing... I will try again :)